pulsar-broker/src/test/java/org/apache/pulsar/broker/service/OneWayReplicatorTest.java

```diff
@@ -57,7 +58,11 @@ public void testReplicatorProducerStatInTopic() throws Exception {
         final byte[] msgValue = "test".getBytes();
 
         admin1.topics().createNonPartitionedTopic(topicName);
-        admin2.topics().createNonPartitionedTopic(topicName);
+        try {
+            admin2.topics().createNonPartitionedTopic(topicName);
+        } catch (PulsarAdminException.ConflictException e) {
+            // Ignore
+        }
```

Comment:
@tisonkun You are right that test code could be improved. I happened to focus on addressing the flakiness without changing the test logic. I'll remove the creation of topics and subscriptions completely since it's not needed in the test.